bindgen-citro3d/src/main.rs

```diff
@@ -0,0 +1,80 @@
+use std::iter::FromIterator;
+use std::path::PathBuf;
+
+use bindgen::callbacks::{DeriveTrait, ImplementsTrait, ParseCallbacks};
+use bindgen::{Builder, RustTarget};
+
+fn main() {
+    let devkitpro = std::env::var("DEVKITPRO").expect("DEVKITPRO not set in environment");
+    let devkitarm = std::env::var("DEVKITARM").expect("DEVKITARM not set in environment");
+
+    let include_path = PathBuf::from(devkitpro).join(PathBuf::from_iter(["libctru", "include"]));
+    let header = include_path.join("tex3ds.h");
+
+    let sysroot = PathBuf::from(devkitarm).join("arm-none-eabi");
+    let system_include = sysroot.join("include");
+
+    let bindings = Builder::default()
+        .header(header.to_str().unwrap())
+        .rust_target(RustTarget::Nightly)
+        .use_core()
+        .trust_clang_mangling(false)
+        .layout_tests(false)
+        .ctypes_prefix("::libc")
+        .prepend_enum_name(false)
+        .fit_macro_constants(true)
+        .raw_line("use ctru_sys::*;")
+        .must_use_type("Result")
+        // TODO functions,types,vars
```

Comment:
Reminder to finish this TODO

Reply:
As it turns out, the default is "all" which amounts to the same thing here, since this is C there are no methods or constructors/destructors to generate. I'll remove the comment